other_processing/0.download-profiles-from-figshare/download.py

```diff
@@ -87,6 +92,6 @@ def download_sqllite_file(filename: Union[str, Path], url: str):
     # initializing parallelization
     n_jobs = len(file_info)
```

Comment:
```suggestion
    n_jobs = len(file_info)
    if n_jobs > mp.cpu_count():
        n_jobs = mp.cpu_count()
```

I may be lacking some knowledge of multiprocessing package here but can't this cause issues if `n_jobs > mp.cpu_count()`? Or maybe there is some other maximum number of processes that should be added here to accommodate all computer setups.

Reply:
Wow... I thought that the `mp.Pool()` was smart enough to default to max number of CPU's if we specify `len(file_info) > max_cpu's` But I guess not, here's a snippet of the [code](https://github.com/python/cpython/blob/af63fa0dcd0d305f6da471779186abc300231893/Lib/multiprocessing/pool.py#L202-L213): 

```python
        if processes is None:
            processes = os.cpu_count() or 1
        if processes < 1:
            raise ValueError("Number of processes must be at least 1")
        if maxtasksperchild is not None:
            if not isinstance(maxtasksperchild, int) or maxtasksperchild <= 0:
                raise ValueError("maxtasksperchild must be a positive int or None")

        if initializer is not None and not callable(initializer):
            raise TypeError('initializer must be a callable')

        self._processes = processes
```

Thanks for pointing this out!